src/Footer/SubmitButton/SubmitButton.js

```diff
@@ -22,7 +23,8 @@ const SubmitButton = ({
   const { state, actions } = useContext(Context)
   const { values } = state
   const { text, prefix, suffix } = button || { text: 'Submit' }
-  const { name, description, isPrint, onSubmit, onSubmitFailed } = global
+  const { name, description, isPrint, onSubmit, onSubmitFailed, resetButton } = global
+  const { resetText , resetEnabled} = resetButton
```

Comment:
also lets add callback into resetButton data
if callback function provided call the function and pass as a argument reset function

Reply:
done